src/client/app/components/meters/EditMeterModalComponent.tsx

```diff
@@ -66,6 +85,89 @@ export default function EditMeterModalComponent(props: EditMeterModalComponentPr
 	const [validMeter, setValidMeter] = useState(isValidMeter(localMeterEdits));
 
 	useEffect(() => { setValidMeter(isValidMeter(localMeterEdits)); }, [localMeterEdits]);
+
+	// Keeps canSave state up to date. Checks if valid and if edit made.
+	// References the original implementation in EditUnitModalComponent.tsx
+	useEffect(() => {
+		// This checks if the inputs for each field is of their appropriate types.
+		// (maxError is left out because it is of type 'any')
+		const validMeter =
```

Comment:
For similar reasons to comment in create conversion, I'm not sure the valid is needed.